tests/losses/sinkpit_wrapper_test.py

```diff
@@ -0,0 +1,150 @@
+import pytest
+import itertools
+import torch
+from torch import nn, optim
+from torch.utils import data
+from torch.testing import assert_allclose
+
+import pytorch_lightning as pl
+from pytorch_lightning import Trainer
+
+from asteroid.losses import PITLossWrapper
+from asteroid.losses import sdr
+from asteroid.losses import singlesrc_mse, pairwise_mse, multisrc_mse
+from asteroid.engine.system import System
+from asteroid.utils.test_utils import DummyWaveformDataset
+
+# target modules
+from asteroid.losses import SinkPITLossWrapper, SinkPITBetaScheduler, sinkpit_default_beta_schedule
+
+
+def bad_loss_func_ndim0(y_pred, y_true):
+    return torch.randn(1).mean()
+
+
+def bad_loss_func_ndim1(y_pred, y_true):
+    return torch.randn(1)
+
+
+def good_batch_loss_func(y_pred, y_true):
+    batch, *_ = y_true.shape
+    return torch.randn(batch)
+
+
+def good_pairwise_loss_func(y_pred, y_true):
+    batch, n_src, *_ = y_true.shape
+    return torch.randn(batch, n_src, n_src)
+
+
+@pytest.mark.parametrize("batch_size", [1, 2, 8])
+@pytest.mark.parametrize("n_src", [2, 5, 10])
+@pytest.mark.parametrize("time", [16000, 1221])
+def test_wrapper(batch_size, n_src, time):
+    targets = torch.randn(batch_size, n_src, time)
+    est_targets = torch.randn(batch_size, n_src, time)
+    for bad_loss_func in [bad_loss_func_ndim0, bad_loss_func_ndim1]:
+        loss = SinkPITLossWrapper(bad_loss_func)
+        with pytest.raises(AssertionError):
+            loss(est_targets, targets)
+
+    loss = SinkPITLossWrapper(good_pairwise_loss_func)
+    loss(est_targets, targets)
+    loss_value, reordered_est = loss(est_targets, targets, return_est=True)
+    assert reordered_est.shape == est_targets.shape
+
+
+@pytest.mark.parametrize("batch_size", [1, 2])
+@pytest.mark.parametrize("n_src", [2, 3, 4])
+@pytest.mark.parametrize("beta,n_iter", [(100.0, 2000)])
+@pytest.mark.parametrize(
+    "function_triplet",
```

Comment:
Maybe import this from the other loss tests instead of redefining it here?

Reply:
How to import locally in tests?

Reply:
Well you can just do `from .loss_functions_test import ...` but it only works if you properly package all the tests (`__init__.py` files etc). I'll resolve this as "not worth the effort" :) A little duplication is okay

Reply:
Agree. Plus, this file is not supposed to test all the loss functions (it doesn't hurt though).